src/main/java/hudson/plugins/sloccount/model/cloc/ClocTotal.java

```diff
@@ -23,6 +23,9 @@
 
     /**
      * Constructor.
+     * @param blank blank
+     * @param comment comment
+     * @param code code
```

Comment:
Comments just to have comments. Not really required

Reply:
The JDK 8 javadoc warnings are not worth fighting.  It is much easier to insert the comments than force yourself to remember on each build that you can ignore that warning.